rudder-agent/debian/rules

```diff
@@ -103,12 +103,9 @@ ifeq ($(findstring systemd,$(WITH)),)
 DEPS := $(DEPS), systemd
 endif
 
-configure:
-	dh_testdir
-	cd SOURCES && ./configure $(WITH)
-
 build:
 	dh_testdir
+	cd SOURCES && ./configure $(WITH)
```

Comment:
why?

Reply:
why not ? :)

Reply:
That's the point of the parent PR : having a configure to call before calling make

Reply:
indeed :D but why moving the configure from the configure target to build?

Reply:
I don't know why the configre target was there first but it doesn't exist in debian specification and thus is never called

Reply:
:man_shrugging: